src/js/components/Header/Header.react.js

```diff
@@ -41,55 +42,26 @@ export default class Header extends Component {
-        const windowControls = this.props.windowControls ? <button className='window-control' onClick={ this.winClose.bind(null) }>&times;</button> : false;
-
-        const volumeClasses = classnames('volume-control', {
-            visible: this.state.showVolume
-        });
-
         return (
             <header className='row'>
                 <Col sm={ 3 } className='main-controls'>
-                    <div className='window-controls'>
-                        { windowControls }
-                    </div>
-
-                    <div className='player-controls'>
-                        <button type='button' className='player-control previous' onClick={ this.previous.bind(null) }>
-                            <Icon name='backward' />
-                        </button>
-                        { playButton }
-                        <button type='button' className='player-control forward' onClick={ this.next.bind(null) }>
-                            <Icon name='forward' />
-                        </button>
-                        <button type='button'
-                                className='player-control volume'
-                                onMouseEnter={ this.showVolume.bind(this) }
-                                onMouseLeave={ this.hideVolume.bind(this) }
-                                onClick={ this.mute }
-                        >
-                            <Icon name={ audio.muted || audio.volume === 0 ? 'volume-off' : audio.volume > 0.5 ? 'volume-up' : 'volume-down' } />
-                            <div className={ volumeClasses }>
-                                <input type={ 'range' }
-                                       min={ 0 }
-                                       max={ 1 }
-                                       step={ 0.01 }
-                                       defaultValue={ Math.pow(audio.volume, 1 / 4) }
-                                       ref='volume'
-                                       onChange={ this.setVolume.bind(this) }
-                                />
-                            </div>
-                        </button>
-                    </div>
+                    <WindowControls
+                        active={ this.props.windowControls }
+                        onClose={ this.winClose }
+                    />
+
+                    <PlayerControls
+                        audio={ Player.getAudio() }
+                        playerStatus={ this.props.playerStatus }
+                        showVolume={ this.state.showVolume }
+                        onPrevious={ this.previous }
+                        onNext={ this.next }
+                        onPlayToggle={ this.playToggle.bind(this) }
+                        onShowVolume={ this.showVolume.bind(this) }
+                        onHideVolume={ this.hideVolume.bind(this) }
+                        onVolumeChange={ this.setVolume.bind(this) }
+                        onMute={ this.mute }
+                    />
```

Comment:
Instead of passing functions to props, just move the functions to `PlayerControls`.
Also, instead of `onPrevious={ this.previous }`, you can refactor to `onPrevious={ AppActions.player.previous }`